src/helpers/drm.ts

```diff
@@ -0,0 +1,57 @@
+import { DRMSystemConfiguration } from "../types";
+
+export const initFairPlayDRM = async (element: HTMLElement, fairPlayOptions: DRMSystemConfiguration) => {
```

Comment:
we can be more precise here and say that we expect `HTMLVideoElement`, mb rename var `element` to `videoElement` ?